tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -57,7 +56,7 @@ def test__get__ok(self):
 
         # Verify Link relationship
         self.assertIsInstance(project.links[0], DbProjectLink)
-        self.assertEqual(project.links[0].id, "test_project_link_id")
+        self.assertCountEqual(project.links[0].id, "test_project_link_id")
```

Comment:
Should this not be `assertEqual`? What prompted the full replacement to `assertCountEqual`? It looks like a looser equality check, and odd for Strings. In particular, the lost requirement on ordering:

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual

Reply:
I accidently changed it to assertCountEqual. Returning to assertEqual